drivers/can/can_esp32_twai.c

```diff
@@ -71,6 +71,7 @@ struct can_esp32_twai_config {
 	const struct device *clock_dev;
 	const clock_control_subsys_t clock_subsys;
 	int irq_source;
+	int irq_priority;
```

Comment:
Picking priority vs level is an effort to steer the DTS in the direction of Zephyr's interrupt system.
Source was unchanged because it would be explicitly wrong to call it irq_number.
We can discuss and change it though, if you disagree.

Reply:
I seem's that irq_priority is more present along the Zephyr source code.